### PR TITLE
Added tests for Oauth and Stubbed tests

### PR DESCRIPTION
**Pull request by NandanAnantharamu**
Oauth test
Stub Mongo DS tests
Updated DS tests by categorising the routes

### CODE REVIEW

File: app/client/cypress/fixtures/testdata.json [hunk 1]
```diff
@@ -115,5 +115,11 @@
   "dropdownErrorMsg": "This value does not evaluate to type \"Array<{ label: string, value: string }>\". Transform it using JS inside '{{ }}'",
   "tableWidgetErrorMsg": "This value does not evaluate to type \"Array<Object>\". Transform it using JS inside '{{ }}'",
   "bindingDataPoint": "{{JSON.stringify(Chart1.selectedDataPoint)}}",
-  "bindChardData": "{{Chart1.selectedDataPoint"
+  "bindChardData": "{{Chart1.selectedDataPoint",
```
Comment (hetunandu) [inline app/client/cypress/fixtures/testdata.json:118]: Spelling mistake
```suggestion
  "bindChartData": "{{Chart1.selectedDataPoint",
```

**Review by hetunandu — CHANGES_REQUESTED** [review]
Minor

**Review by hetunandu — DISMISSED** [review]

**Review by hetunandu — APPROVED** [review]

File: app/client/cypress/fixtures/testdata.json (hunk 1 above)
Comment (NandanAnantharamu) [reply app/client/cypress/fixtures/testdata.json]: this is resolved

**Review by hetunandu — APPROVED** [review]

File: app/client/cypress/support/commands.js [hunk 2]
```diff
@@ -96,34 +94,39 @@ Cypress.Commands.add("CheckShareIcon", (orgName, count) => {
   ).should("have.length", count);
 });
 
+Cypress.Commands.add("stubPostHeaderReq", () => {
+  cy.intercept("POST", "/api/v1/users/invite", (req) => {
+    headers: {
+      accept: "application/json";
```
Comment (hetunandu) [inline app/client/cypress/support/commands.js:100]:
```suggestion
      accept: "application/json";
      origin: "dev.appsmith.com";
```

**Review by hetunandu — APPROVED** [review]

**Review by hetunandu — APPROVED** [review]

**Pull request merged**